src/Auth0Client.ts

```diff
@@ -482,9 +495,9 @@ export default class Auth0Client {
       throw new Error('Invalid state');
     }
 
-    if (error) {
-      this.transactionManager.remove();
+    this.transactionManager.remove();
```

Comment:
I simply moved this out of the `if (error)` block and removed the duplicate call.